packages/lodestar/src/sync/sync.ts

```diff
@@ -119,10 +119,8 @@ export class BeaconSync implements IBeaconSync {
     if(this.mode === SyncMode.STOPPED) return;
     this.mode = SyncMode.REGULAR_SYNCING;
     await this.initialSync.stop();
-    await Promise.all([
-      this.gossip.start(),
-      this.regularSync.start()
-    ]);
+    await this.regularSync.start();
```

Comment:
should we swap: make `await this.gossip.start();` first?

Reply:
perhaps we can revert to the previous Promise.all()